src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started digging");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped diggind");

```

Comment:
There is a typo in the message 'Excavator stopped diggind'. It should be 'digging'. Moreover, this message should also be more informative by specifying that it is the Excavator machine that has stopped working.